src/components/Picker/Picker.tsx

```diff
@@ -408,8 +412,13 @@ class Picker extends React.PureComponent<Props, State> {
       onBlur: autoSuggestMethods.onBlur,
       disabled: readOnly || disabled || (!!this.props.maxSelectedItems && this.props.maxSelectedItems <= chipListState.length),
     };
+<<<<<<< HEAD
```

Comment:
@Farhan-emgage it appears a merge conflict was included in this PR.

Reply:
@weegrom The conflict is resolved